settings.gradle

```diff
@@ -27,8 +28,10 @@ buildscript {
     }
 }
 
+def isJBFork = true
```

Comment:
I guess it's a minor thing since these changes won't be upstreamed:
is it possible to have one common `isJBFork` value? I see another one above on line 8.

Reply:
I tried to define it in `buildSrc`, but it seems it isn't possible to use it inside settings.gradle

Reply:
got it